NEWS.md

```diff
@@ -1,18 +1,102 @@
-3.31.2
+3.32.1
 ======
 
+## Highlights
+
+ * `mongodb()`: add `workers()` support (multi-threaded connection pooling)
+   
+   The MongoDB driver now supports the `workers()` option, which specifies the
+   number of parallel workers to be used.
+   Workers are based on the connection pooling feature of the MongoDB C library.
+   
+   This increases the throughput of the MongoDB destination driver.
+   
+   Example:
+   
+   ```
+   destination {
+     mongodb(
+       uri("mongodb://hostA,hostB/syslog?replicaSet=my_rs&wtimeoutMS=10000&socketTimeoutMS=10000&connectTimeoutMS=10000&serverSelectionTimeoutMS=5000")
+       collection("messages")
+       workers(8)
+     );
+   };
+   ```
+   ([#3621](https://github.com/syslog-ng/syslog-ng/pull/3621))
+ * `mongodb()`: template support for the `collection()` option
+   
+   The `collection()` option of the MongoDB destination driver now accepts
+   templates, for example:
+   
+   ```
+   destination {
+     mongodb(
+       uri("mongodb://host/syslog?wtimeoutMS=10000&socketTimeoutMS=10000&connectTimeoutMS=10000&serverSelectionTimeoutMS=5000")
```

Comment:
In the `collection()` example, the URI is unnecessarily long. It was me, sorry.
Could we change the second example something shorter, for example:

```
destination {
  mongodb(
    uri("mongodb://host/syslog")
    collection("${HOST}_messages")
    workers(8)
  );
};
```